parts/k8s/kubernetesinstalls.sh

```diff
@@ -278,7 +278,7 @@ installKubeletAndKubectl() {
 pullContainerImage() {
     CLI_TOOL=$1
     DOCKER_IMAGE_URL=$2
-    if [ ! -z "${PRIVATE_AZURE_REGISTRY_SERVER}" ]; then
+    if [[ ! -z "${PRIVATE_AZURE_REGISTRY_SERVER:-}" ]]; then
```

Comment:
double `[[` and `]]` just because it's more idiomatic w/ the rest of the bash surface area